stable/metabase/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: The easy, open source way for everyone in your company to ask questions
   and learn from data.
 name: metabase
-version: 0.12.0
-appVersion: v0.35.3
+version: 0.12.1
+appVersion: v0.36.3
```

Comment:
Shouldn't patch reset to 0 if minor is bumped?

Reply:
You mean the appVersion ?
If so then yes, of course there is v0.36.0, v0.36.1, v0.36.2, ... and at the time of this PR v0.36.3 is the latest version.